add_to_app/flutter_module_books/lib/main.dart

```diff
@@ -0,0 +1,191 @@
+import 'package:flutter/material.dart';
+// This file is autogenerated from the schema file at pigeon/schema.dart.
+// Regenerate by running (from flutter_module_books):
+// flutter pub run pigeon \
+//     --input pigeon/schema.dart \
+//     --java_out ../android_books/app/src/main/java/dev/flutter/example/books/Api.java \
+//     --java_package "dev.flutter.example.books"
+import 'package:flutter_module_books/api.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: ThemeData(
+        primaryColor: const Color(0xff6200ee),
```

Comment:
So that's the new material swatch colors. We don't have it in Flutter yet. i.e. it's not one of the Colors. It's in ColorSwatch now but widgets aren't reading it yet (like the AppBar which is what I want).